homeassistant/components/jewish_calendar/strings.json

```diff
@@ -1,4 +1,15 @@
 {
+  "entity": {
+    "sensor": {
+      "hebrew_date": {
+        "state_attributes": {
+          "year": { "name": "Year" },
+          "month_name": { "name": "Month Name" },
+          "day": { "name": "Day" }
```

Comment:
Should it be more clear this is year/month/day according to the Hebrew calendar, or is that immediately clear to the user?

Reply:
Done